### PR TITLE
feat(webpack): Use webpack to reduce extension loading time

### PR DESCRIPTION
**Pull request by arajkumar**
This PR fixes https://github.com/fabric8-analytics/fabric8-analytics-vscode-extension/issues/287, https://github.com/fabric8-analytics/fabric8-analytics-vscode-extension/issues/368

- vsix file size:
   - before webpack: (2532 files, 3.39MB)
   - after webpack: (15 files, 371.99KB)
   - file reduced by: (9 times)
   - total files reduced by:( 168 times)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -188,21 +190,24 @@
...
-    "glob": "^7.1.3",
-    "istanbul": "^0.4.5",
-    "mocha-jenkins-reporter": "^0.4.0",
-    "remap-istanbul": "^0.12.0"
+    "webpack": "^4.40.1",
+    "webpack-cli": "^3.3.8"
   },
   "dependencies": {
-    "fabric8-analytics-lsp-server": "0.1.56",
+    "fabric8-analytics-lsp-server": "file:../fabric8-analytics-lsp-server/output/fabric8-analytics-lsp-server-0.0.0-development.tgz",
```
Comment (arajkumar) [inline package.json]: This is a temp fix intended for dev purpose, will be removed once we get latest lsp module.

**Comment by arajkumar** [comment]
vsix size is ~535K, it was ~2.5MB prior to this fix.

**Review by dvandra — APPROVED** [review]
LGTM

File: test/runTest.ts [hunk 2]
```diff
@@ -13,9 +13,9 @@ async function main() {
         const extensionTestsPath = path.resolve(__dirname, './');
 
         // Download VS Code, unzip it and run the integration test
-        await runTests({version: '1.44.1', extensionDevelopmentPath, extensionTestsPath, launchArgs: [path.resolve(__dirname, '../../test/resources/sampleNodeApp/')] });
+        await runTests({extensionDevelopmentPath, extensionTestsPath, launchArgs: [path.resolve(__dirname, '../../test/resources/sampleNodeApp/')] });
```
Comment (mathur07) [inline test/runTest.ts:16]: Is it possible that not defining a version may create a unknown failure of test cases in future?
Or Mocha will take care of it?

Comment (arajkumar) [reply test/runTest.ts:16]: Nope, it wasn't new enough to run the latest webpacked js., so just removed the version pinning. I will find a version which works for case. It is a good catch 👍

Comment (arajkumar) [reply test/runTest.ts]: 1.46.1 seems to be working.

**Comment by arajkumar** [comment]
[test]

File: test/runTest.ts (hunk 2 above)
Comment (mathur07) [reply test/runTest.ts]: Okay, Cool :)

**Review by mathur07 — APPROVED** [review]

**Pull request merged**